cmd/ci-operator/main.go

```diff
@@ -712,6 +712,8 @@ func (o *options) Complete() error {
 		return err
 	}
 
+	applyEnvOverrides(o)
```

Comment:
This should be placed before:
```golang
if err := overrideMultiStageParams(o); err != nil {
    return err
}
```
I guess cmd arguments take precedence over the env. variables.
One drawback of keeping `overrideMultiStageParams` and `applyEnvOverrides` separate is that now is quite hard to write a unit test to makes sure that cmd args override env. variables.
Unfortunately `func (o *options) Complete() error` is quite a mess to test.

Reply:
I adjusted the order of applyEnvOverrides and handleTargetAdditionalSuffix yesterday, but the E2E tests failed, so I reverted the order of these two functions.